src/components/__tests__/TextArea.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { shallowMount, Wrapper } from "@vue/test-utils";
+import TextArea from "@/components/TextArea.vue";
+
+describe("TextArea.vue", () => {
+  let wrapper: Wrapper<TextArea>;
+  let vm: any;
+
+  beforeEach(() => {
+    wrapper = shallowMount(TextArea, {
+      propsData: {
+        placeholder: "Type text..",
+      },
+    });
+    vm = wrapper.vm;
+  });
+
+  it("does a wrapper exist", () => {
+    expect(wrapper.isVueInstance).toBeTruthy();
+  });
+
+  it("check class of TextArea", () => {
+    expect(wrapper.classes()).toContain("textarea");
+  });
+
+  it("check placeholder", () => {
+    expect(wrapper.attributes("placeholder")).toBe("Type text..");
+  });
+
+  it("should set isReadonly", async () => {
+    expect(vm.isReadonly).toBeFalsy();
```

Comment:
```suggestion
    expect(
      wrapper.find("textarea").element.getAttribute("readonly")
    ).toBeFalsy();
```

Reply:
or `wrapper.element` as you use in other tests

this test was too simple
It sets the prop and check this prop 
let's rather test the final result of changing this prop